src/components/NewBeers.jsx

```diff
@@ -0,0 +1,140 @@
+import React, { Component } from 'react';
+import Header from './Header';
+import axios from 'axios';
+import { Link } from 'react-router-dom';
+
+export class NewBeers extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: '',
+      tagline: '',
+      description: '',
+      first_brewed: '',
+      brewers_tips: '',
+      attenuation_level: 0,
+      contributed_by: '',
+    };
+  }
+
+  handleSubmit = (evento) => {
+    evento.preventDefault();
+    // const body = this.state;
+
+    axios
+      .post('https://ih-beers-api2.herokuapp.com/beers/new', this.state)
+      .then((res) =>
+        console.log(res.data)(
+          (this.setState = {
```

Comment:
Si te falla la API aca puede ser por que le falte algun parametro cuando le mandas la llamada,

Aun asi aca tenes un error de syntaxis.

this.setState = .... ????

this.setState({ .... })

otra cosa, tenes this.setState envuelto en ( ), no se por que ( this.setState = { ...} ), eso tampoco deberia de ir